### ISSUE TITLE
How to define Output Dimension in NCP/LTC network?

### ISSUE THREAD

**Issue by Mariam-124** [mlech26l/ncps#59]
Hi , I am trying to use a multivariant LST model for stock price prediction , and that model defines output dimension which is different from input dimension.
But I am unable to understand how to add output dimension in NCP.

Previous model:
_model = Sequential()
model.add(LSTM(64, activation='relu', input_shape=(trainX.shape[1], trainX.shape[2]), return_sequences=True))
model.add(LSTM(32, activation='relu', return_sequences=False))
model.add(Dropout(0.2))
model.add(Dense(trainY.shape[1]))_ 

My LTC Model is

wiring = wirings.AutoNCP(8,1) # 8 neurons in total, 1 output (motor neuron)
model = keras.models.Sequential(
    [
        keras.layers.InputLayer(input_shape=(None, 5)),
        # here we could potentially add layers before and after the LTC network
        LTC(wiring, return_sequences=True),
    ]
)
model.compile(
    optimizer=keras.optimizers.Adam(0.01), loss='mean_squared_error'
)

model.summary()
